test/model/test_string_constraints.py

```diff
@@ -41,6 +41,25 @@ def test_version_type(self) -> None:
         version = "0"
         _string_constraints.check_version_type(version)
 
+    def test_aasd_130(self) -> None:
+        name: model.NameType = "\0"
```

Comment:
Can you please add a docstring what AASd-130 says, so that we do not have to look into the specification each time we read it?